aacparser.go

```diff
@@ -891,7 +891,7 @@ func (adts *ADTS) adts_header_error_check() {
 
 	if !adts.protection_absent {
 		data.Raw_data_block_position = make([]uint16, adts.num_raw_data_blocks)
-		for i := uint8(1); i <= adts.num_raw_data_blocks; i++ {
+		for i := uint8(1); i < adts.num_raw_data_blocks; i++ {
```

Comment:
The spec also specifically spells out this indexing method in 1.A.9.  This fix will result in potentlaly missing the last `data_block`.  We need to do some investigating to see if 0 indexing this makes sense and it's actually a typo in the spec.  Can you look into the source for `libav` from ffmpeg and/or `libfdkaac` and/or `faad`?  Those libraries have a much higher chance of parsing real input around this rather than fuzzed input.